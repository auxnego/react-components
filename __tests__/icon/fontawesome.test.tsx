import React from "react";
import {create} from "react-test-renderer";

import FontAwesome from "../../icon";

it("check icon", () => {

  expect(create(
    <FontAwesome to="check" />
  ).toJSON()).toMatchSnapshot();

});

it("check icon and changing base", () => {

  expect(create(
    <FontAwesome to="check" base="fas"/>
  ).toJSON()).toMatchSnapshot();

});

it("check icon and changing setEl", () => {

  expect(create(
    <FontAwesome to="check" setEl="div"/>
  ).toJSON()).toMatchSnapshot();

});
