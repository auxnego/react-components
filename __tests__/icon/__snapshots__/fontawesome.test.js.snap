// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`check icon 1`] = `
<i
  className="fa fa-check"
/>
`;

exports[`check icon and changing base 1`] = `
<i
  className="fas fa-check"
/>
`;

exports[`check icon and changing setEl 1`] = `
<div
  className="fa fa-check"
/>
`;
